src/utils.ts

```diff
@@ -0,0 +1,4 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+export const PORT = parseInt(String(process.env.PORT), 10);
```

Comment:
Is it worth making this fail more explicitly if PORT isn't set, or set a sensible default?

Reply:
I've added the validation library for env vars in nodejs